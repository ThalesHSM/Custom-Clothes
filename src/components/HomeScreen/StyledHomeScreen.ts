import styled from 'styled-components';

export const StyledMainDiv = styled.div`
  display: flex;
  flex: 1;
  flex-direction: column;
`;
