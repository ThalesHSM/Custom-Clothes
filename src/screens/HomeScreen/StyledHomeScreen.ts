import styled from 'styled-components';

export const StyledMainDiv = styled.div`
  display: flex;
  flex: 1;
  flex-direction: column;
  @import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&display=swap');
  font-family: 'Lato', sans-serif;
`;
