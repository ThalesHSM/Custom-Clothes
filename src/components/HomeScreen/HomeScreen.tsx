import React, { useState } from 'react';
import Header from '../header/Header';
import { StyledMainDiv } from './StyledHomeScreen';

function HomeScreen() {
  const [isLoginMenuOpen, setIsLoginMenuOpen] = useState(false);
  return (
    <StyledMainDiv>
      <Header setIsLoginMenuOpen={setIsLoginMenuOpen} />
      {isLoginMenuOpen ? <p id="test">login</p> : null}
    </StyledMainDiv>
  );
}

export default HomeScreen;
