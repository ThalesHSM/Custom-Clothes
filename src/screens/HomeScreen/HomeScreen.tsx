import React, { useState } from 'react';
import Header from '@Components/header/Header';
import Clothes from '@Components/clothes/Clothes';
import SideMenu from '@Components/sideMenu/SideMenu';
import Footer from '@Components/footer/Footer';

import { StyledMainDiv } from './StyledHomeScreen';

function HomeScreen() {
  const [isLoginMenuOpen, setIsLoginMenuOpen] = useState(false);

  return (
    <StyledMainDiv>
      <Header
        isLoginMenuOpen={isLoginMenuOpen}
        setIsLoginMenuOpen={setIsLoginMenuOpen}
      />

      <div>{isLoginMenuOpen ? <p id="test">login</p> : null}</div>

      <div style={{ display: 'flex' }}>
        <SideMenu />

        <Clothes />
      </div>
      <Footer />
    </StyledMainDiv>
  );
}

export default HomeScreen;
