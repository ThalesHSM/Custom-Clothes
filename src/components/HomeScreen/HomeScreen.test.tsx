import React from 'react';
import { shallow, mount } from 'enzyme';
import HomeScreen from './HomeScreen';

describe('MyComponent', () => {
  let component;

  beforeEach(() => {
    component = shallow(<HomeScreen />);
  });

  it('should render the HomeScreen', () => {
    expect(component).toMatchSnapshot();
    component.unmount();
  });
});
