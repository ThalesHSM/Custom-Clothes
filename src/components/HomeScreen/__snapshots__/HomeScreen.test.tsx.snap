// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render the HomeScreen 1`] = `
<styled.div>
  <Header
    setIsLoginMenuOpen={[Function]}
  />
</styled.div>
`;
